engine/src/test/scala/engine/backend/jes/JesBackendCallSpec.scala

```diff
@@ -1,21 +1,17 @@
 package cromwell.engine.backend.jes
 
-import java.net.URL
-
+import akka.actor.ActorSystem
 import cromwell.engine.WorkflowDescriptor
 import cromwell.engine.backend.BackendCallJobDescriptor
+import cromwell.engine.backend.runtimeattributes.CromwellRuntimeAttributes
 import cromwell.engine.workflow.BackendCallKey
 import org.scalatest.{FlatSpec, Matchers}
-import org.slf4j.{Logger, LoggerFactory}
 import org.specs2.mock.Mockito
 import wdl4s.CallInputs
 
+/** FIXME this class references a JesBackendCall which is soon to disappear, move all this to JesBackendSpec. */
```

Comment:
You might want to take a look at #516, that removes `BackendCall` but this code was basically untouched since actual references to `BackendCall` were already removed here.  :frowning:

The reason I did it this way was so reviewers could judge with a minimum of noise if the assertions that are now being made are equivalent to those that were being made on `BackendCall`s.